addon/ng2/blueprints/ng2/files/src/app.ts

```diff
@@ -1,4 +1,4 @@
-import {bootstrap} from 'angular2/angular2';
+import {bootstrap} from 'angular2/platform/browser';
```

Comment:
above this line add:

```
/// <reference path="node_modules/angular2/manual_typings/globals-es6.d.ts"/>
```

this will resolve all the `tsc` issues - this is a bug in angular a47. I'll file an issue.

let's add this line and we'll remove it once the bug is fixed.